libminifi/src/utils/net/Socket.cpp

```diff
@@ -14,19 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 #include "utils/net/Socket.h"
-#include "Exception.h"
-#include <cstring>
```

Comment:
`<cstring>` is still used for `memcpy`.

Reply:
'<cstring>` is still there it was only moved a bit lower